modules/book-list.js

```diff
@@ -0,0 +1,52 @@
+import { addBook, removeBook } from './bookUtils.js';
+
+class BookList {
+  constructor() {
+    this.books = [];
+    this.addBook();
+    this.loadBooks();
+  }
+
+  addBook = () => {
+    addBook(this);
+  }
+
+  loadBooks = () => {
+    const storedBooks = localStorage.getItem('books');
+    if (storedBooks) {
+      this.books = JSON.parse(storedBooks);
+      this.books.forEach((book) => {
+        this.displayBook(book);
+      });
+    }
+  }
+
+  displayBook = (book) => {
+    const bookList = document.getElementById('booklist');
+    const table = document.createElement('table');
+    table.classList.add('table', 'table-striped');
+    table.innerHTML = `
+        <tr>
+        <td>
+          "${book.title}" by 
+          ${book.author}
+          </td>
+          <td>
+          <button type="button" class="remove-btn">Remove</button></td>
+        </tr>
+      `;
+    bookList.appendChild(table);
+
+    const removeButton = table.querySelector('.remove-btn');
+    removeButton.addEventListener('click', () => {
+      this.removeBook(book);
+      table.remove();
+    });
+  }
+
+  removeBook = (book) => {
+    removeBook(this, book);
+  }
+}
```

Comment:
- [ ] Great job working on some changes requested by previous reviewer. We still have some unresolved issues
As indicated by previous reviewer
 ```
- The whole point of this milestone is to divide our code into modules. While you have made some effort to modularize the code, we can improve.
 - Consider putting each of these functions in separate .js files for a more modular approach. For example, you could just call the addBook and removeBook functions by importing them.
 - You could also have a separate module for LocalStorage and you can just call that function when you need to save or retrieve data in your app.
```

Please kindly fix your structure as requested